frontend/src/modules/Traces/LatencyModalForm.tsx

```diff
@@ -44,14 +45,38 @@ const LatencyModalForm: React.FC<LatencyModalFormProps> = ({
 					<Col span={12}>
 						<Form.Item
 							name="min"
-							label="Min (in ms)"
+                            label="Min (in ms)"
+							rules={[
+								({ getFieldValue }) => ({
```

Comment:
Declare this block of code above and use it here. This will help in code readability.

Reply:
Hey, @nidhi-tandon I didn't understand your comment, Any examples that you can share so that I can understand ?

Reply:
@saideepesh000  

If I understand it correctly, you have to just move the inline function and create separate function and use it here. Something like this:

```js
const validateMaxValues = ({ getFieldValue }) => ({
  validator(_, value) {
     if (value > getFieldValue('min')) {...}
     ...
  }
})
```

```
<Form.Item rules={[validateMaxValues()]} />
```